lib/src/main/java/com/fabernovel/statefullayout/StatefulLayout.kt

```diff
@@ -181,28 +192,28 @@ class StatefulLayout : FrameLayout, StateContainer<Int, State> {
      * @throws [NoSuchElementException] if [id] was not found.
      */
     override fun showState(@IdRes id: Int): State {
-        return showState(id, true)
+        return showState(id, areTransitionsEnabled)
```

Comment:
Just to be sure, you can't use kotlin default param syntax to support java usage of this class, right ?

Reply:
I didn't want to introduce a breaking change.
I forgot about those, I'll fix it.

Reply:
I can't because there's the `StateContainer` interface which don't have the boolean.